app/views/assignments/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<p id="notice"><%= notice %></p>
```

Comment:
Just wanted to double check that 'notice' is supposed to be in erb brackets, or is this just text?